admin/weaviate/README.md

```
@@ -0,0 +1,305 @@
+# Weaviate RBAC Setup for Edutelligence
+
+This directory contains scripts for setting up Role-Based A### Iris Service Connection
+### Athena Service Connection
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)